bellows/uart.py

```diff
@@ -249,9 +249,11 @@ def _unstuff(self, s):
 
 
 @asyncio.coroutine
-def connect(port, application, loop=None):
+def connect(port, application, loop=None, baudrate=None):
     if loop is None:
         loop = asyncio.get_event_loop()
+    if baudrate is None:
```

Comment:
But I'm passing in "None" from the cli and I'm always assigning the baudrate, so we will never use the default value, because it is always overwritten.
Should I try to rework the other layers so that they check if the baudrate is None, before calling the uart connect?